lib/gettext-setup/gettext_setup.rb

```diff
@@ -37,6 +36,7 @@ def self.initialize(locales_path = 'locales', options = {})
     FastGettext.default_text_domain = 'master_domain'
 
     # Likewise, be explicit in our default language choice.
+    FastGettext.default_available_locales = [] if FastGettext.default_available_locales == true
```

Comment:
In what circumstances might this be set to `true`?

Reply:
I see the comment in the issue about `TrueClass`, but I don't understand it.

Reply:
Try running the spec tests, that's what happens.

Reply:
I can't run the spec tests because it has dependencies on external programs instead of stubbing that out.

Reply:
So the problem is because `nil | []` returns true. I'll put up a PR to fix this issue.

Reply:
Ah, nice, I was trying to track that down on Thursday.